Makefile

```diff
@@ -559,6 +559,7 @@ live: $(LIVE_IMG) $(BIOS_IMG) current	; $(QUIET): "$@: Succeeded, LIVE_IMG=$(LIV
 live-%: $(LIVE).%		; $(QUIET): "$@: Succeeded, LIVE=$(LIVE)"
 installer: $(INSTALLER_IMG)
 installer-%: $(INSTALLER).% current ; @echo "$@: Succeeded, INSTALLER_IMG=$<"
+generate_metadata: $(GENERATE_METADATA)
```

Comment:
Is there any reason to call this `generate_metadata` and not something like `sources`?

Reply:
No specific reason, this popped up while I was working. Open to suggestions. sources would be bit mis-leading. I will think of a better name.

Reply:
Do you think something like collect_sources would be misleading?

Reply:
collect_sources sounds good. I was thinking to use `package_info` - what is your opinion for the same ?

Reply:
But it isn't about the info; it is about all of the source files.

Reply:
will keep `collect_sources`